clients/unshaded/src/main/java/tachyon/client/file/FileOutStream.java

```diff
@@ -204,6 +204,7 @@ public void write(int b) throws IOException {
 
   @Override
   public void write(byte[] b) throws IOException {
+    Preconditions.checkArgument(b != null, PreconditionMessage.ERR_WRITE_BUFFER_NULL);
```

Comment:
We won't call the other write method if `b == null` because `b.length` will throw `NullPointerException`.